instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -35,23 +35,7 @@
   private static final Logger log = LoggerFactory.getLogger(Config.class);
   private static final Pattern PROPERTY_NAME_REPLACEMENTS = Pattern.compile("[^a-zA-Z0-9.]");
 
-  // locations where the context was propagated to another thread (tracking multiple steps is
-  // helpful in akka where there is so much recursive async spawning of new work)
-  public static final Context.Key<List<StackTraceElement[]>> THREAD_PROPAGATION_LOCATIONS =
-      Context.key("thread-propagation-locations");
-  public static final boolean THREAD_PROPAGATION_DEBUGGER =
-      Boolean.getBoolean("otel.threadPropagationDebugger");
-
-  public static final String DEFAULT_EXPORTER = "otlp";
-  public static final boolean DEFAULT_TRACE_ENABLED = true;
-  public static final boolean DEFAULT_INTEGRATIONS_ENABLED = true;
-  public static final boolean DEFAULT_RUNTIME_CONTEXT_FIELD_INJECTION = true;
-  public static final boolean DEFAULT_TRACE_EXECUTORS_ALL = false;
-  public static final boolean DEFAULT_SQL_NORMALIZER_ENABLED = true;
-  public static final boolean DEFAULT_KAFKA_CLIENT_PROPAGATION_ENABLED = true;
-  public static final boolean DEFAULT_HYSTRIX_TAGS_ENABLED = false;
-
-  private static final Config DEFAULT = Config.newBuilder().build();
+  private static final Config DEFAULT = Config.create(Collections.emptyMap());
```

Comment:
So default config is one without any configuration? Maybe it should use old default values in some way?

Reply:
The default values are now specified by `Config` callers, so there's no need to keep anything here -- which makes sense, because we wouldn't want to keep a default value for some Kafka property here.